consensus/polybft/polybft.go

```diff
@@ -131,7 +131,7 @@ func GenesisPostHookFactory(config *chain.Chain, engineName string) func(txn *st
 		}
 
 		if err = initContract(contracts.ValidatorSetContract, input, "ChildValidatorSet", transition); err != nil {
-			return err
+			return fmt.Errorf("ChildValidatorSet SC call failed! error: %w", err)
```

Comment:
Nitpick: I would consider slightly changing this error message. Also similar should be done for `NativeTokenContract`. https://github.com/0xPolygon/polygon-edge/blob/df4e5dae55b2f32c6d7080d43f0c469b3248907d/consensus/polybft/polybft.go#L143

```suggestion
			return fmt.Errorf("ChildValidatorSet SC initialization failed: %w", err)
```

Reply:
Reworked the docker scripts, so that these code changes are not needed. 
Reverted all the code changes, so now only docker related files are changed.